modules/utxo-lib/src/bitgo/zcash/ZcashPsbt.ts

```diff
@@ -7,9 +7,19 @@ import {
 import { Network, PsbtTransaction } from '../../';
 import { Psbt as PsbtBase } from 'bip174';
 import * as types from 'bitcoinjs-lib/src/types';
-
+import { UtxoTransaction } from '../UtxoTransaction';
 const typeforce = require('typeforce');
 
+enum Subtype {
+  ZEC_CONSENSUS_BRANCH_ID = 0x00,
+}
+const CONSENSUS_BRANCH_ID_KEY = Buffer.concat([
+  Buffer.of(0xfc),
+  Buffer.of(0x05),
+  Buffer.from('BITGO', 'ascii'),
+  Buffer.of(Subtype.ZEC_CONSENSUS_BRANCH_ID),
+]);
+
```

Comment:
Nice. When we have more keys in here (eg. for musig2-x-only-agg) we'll probably have a function to relate the enum to the buffer key, but that is future-our problem.